src/doc/gui_manual/Quantitative/LineSampler.rst

```diff
@@ -0,0 +1,331 @@
+
+.. _LineSampler_ListTab:
+
+.. figure:: images/LineSamplerListTab.png
+
+   LineSampler List Tab
+
+
+Sampling Tab
+""""""""""""
+
+The sampling tab sets up how each channel will be sampled.
+
+Geometry
+
+1. Channel geometry - Currently the sampling geometry is limited to a
+   point or along a line. Future plans are to have cylindrical and
+   cone sampling geometries.
+
+2. Linear sampling distance - Sample along each channel using the
+   distance specified.
+
+3. Sample volume - For each sample multiple it by a volume.
+
+4. Channel radius - The radius of a channel that is describe by a
+   cylinder.
+
+5. Sample profile - The sample profile of a channel that is describe
+   by a cylinder. Either a TopHat or Gaussian profile. If a Gaussian
+   profile is selected the standard deviation may be given.
+
+6. Cone divergence - For a cone the divergence of the channel.
+
+Integration
+
+7. Channel integration - When sampling one can sample along the
+   channel recording each indivdual sample or integrate (sum) all of
+   the sample values togethere.
+
+
+8. Toroidal integration - When sampling toroidally one can sample
+   along the circumference recording each sample or integrate all of
+   the sample values together.
+
+Toroidal angle sampling
+
+9. Sample - When sampling toroidally one can sample relative to the
+start point or on an absolute basis.
+
+10. Toroidal sample angle - The start, stop, and stride for toroidal
+    sampling.
+
+Time sampling
+
+11. When sampling one can sample just the current time step or across
+multiple times steps which becomes the X axis.
+
+12. Time step - The start, stop, and stride for time sampling.
+
+.. _LineSampler_SamplingTab:
+
+.. figure:: images/LineSamplerSamplingTab.png
+
+   LineSampler Sampling Tab
+
+
+View Tab
+""""""""
+
+The 'View' tab sets attributes based on the dimension of the plot.
+
+1. When associating the LineSampler operator with a specific plot, the
+   operator needs to know the plot's view dimension so to display the
+   sample data correcly. Normally one would have three windows, 1D,
+   2D, and 3D. The Line Sampler operator would be active for the three
+   plots in each window and one would individually set this attribute
+   for each.
+
+2. When checked, assures that when the operator attributes are updated
+   that the view dimension is not updated to all plots. Should always
+   be set to true.
+
+When displaying the resulting sampling as a 1D plot various viewing
+prarmeters can be set.
+
+3. Scale each channel's Y value.
+
+4. For each channel offset the Y value, so that possibly overlapping
+   channels are offset.
+
+5. For each array offset the X value, so that possibly overlapping
+   arrays are offset.
+
+6. When sampling over time the set the X axis to be either the Step,
+   Time, or Cycle.
+
+
+7. The view geometry can be restricted to being Points, Lines, or
+   Surfaces in much same way one can do in the Pseudocolor Plot.
```

Comment:
This is more a question of implementation than your documentation:
At the beginning you state that LineSampler needs to be used in conjunction with Pseudocolor plot. If that is the case, why the duplication of functionality in restricting the geometry to Points/Lines/Surfaces? Is the Pseudocolor plot implementation of this not sufficient? If the plot options are set differently than the operator, which will take precedence?

Reply:
I need to investigate as this functionality is probably a hold over from before your changes to the Pseudocolor plot.